compiler/rustc_codegen_cranelift/scripts/filter_profile.rs

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
-#![forbid(unsafe_code)]/* This line is ignored by bash
+#![forbid(unsafe_code)]
+/* This line is ignored by bash
```

Comment:
Please revert this change. It is necessary for being both valid rust code and valid bash script.

Reply:
oh huh... rustfmt changed this.

Reply:
It is in the ignore section of `rustfmt.toml`, but rust-analyzer doesn't respect this.